school/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :id, :name, :parent_permission, :age
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] It appears that the `id` instance variable is not being initialized in the constructor. To ensure that each person has a unique ID, I recommend generating a value using the code snippet provided below:
```rb
@id = Random.rand(1..1000)
```